### PR TITLE
Add capella fork epoch for Goerli

### PR DESCRIPTION
**Pull request by terencechain**
Reference: https://github.com/eth-clients/goerli/pull/160/

### CODE REVIEW

**Comment by nisdas** [comment]
I think we should wait until that PR is merged in.

**Comment by potuz** [comment]
That PR also changes SHARDING for DENEB, shouldn't this affect our yaml files?

**Comment by terencechain** [comment]
> That PR also changes SHARDING for DENEB, shouldn't this affect our yaml files?

I think we already changed that

**Review by nisdas — APPROVED** [review]

**Pull request merged**
